wiki/Tournaments/TWC/2018/en.md

```diff
@@ -77,278 +75,274 @@ The osu!taiko World Cup 2018 is run by various community members by distributing
 
 ### Semifinals
 
-| Saturday, 2018-04-28 | | | | |
-| ---: | :---: | :---: | :--- | :---: |
-| Australia ![][flag_AU] | 6 | 2 | ![][flag_CL] Chile | [#1](https://osu.ppy.sh/community/matches/42025926) |
-| Brazil ![][flag_BR] | 0 | 6 | ![][flag_ID] Indonesia | [#1](https://osu.ppy.sh/commuity/matches/42028951n) |
-| Poland ![][flag_PL] | 2 | 6 | ![][flag_ES] Spain | [#1](https://osu.ppy.sh/community/matches/42032146) |
+| Saturday, 2018-04-28 |  |  |  |  |
+| --: | :-: | :-: | :-- | :-: |
+| **Australia** ![][flag_AU] | **6** | 2 | ![][flag_CL] Chile | [#1](https://osu.ppy.sh/community/matches/42025926) |
+| Brazil ![][flag_BR] | 0 | **6** | ![][flag_ID] **Indonesia** | [#1](https://osu.ppy.sh/commuity/matches/42028951n) |
+| Poland ![][flag_PL] | 2 | **6** | ![][flag_ES] **Spain** | [#1](https://osu.ppy.sh/community/matches/42032146) |
 
-| Sunday, 2018-04-29 | | | | |
-| ---: | :---: | :---: | :--- | :---: |
-| Hong Kong ![][flag_HK] | 5 | 6 | ![][flag_CA] Canada | [#1](https://osu.ppy.sh/community/matches/42048757) |
-| United States ![][flag_US] | 3 | 6 | ![][flag_TW] Taiwan | [#1](https://osu.ppy.sh/community/matches/42050275) |
-| Japan ![][flag_JP] | 6 | 0 | ![][flag_KR] South Korea | [#1](https://osu.ppy.sh/community/matches/42051882) |
-| Indonesia ![][flag_ID] | 6 | 2 | ![][flag_AU] Australia | [#1](https://osu.ppy.sh/community/matches/42061047) |
-| Canada ![][flag_CA] | 6 | 1 | ![][flag_ES] Spain | [#1](https://osu.ppy.sh/community/matches/42069729) |
+| Sunday, 2018-04-29 |  |  |  |  |
+| --: | :-: | :-: | :-- | :-: |
+| Hong Kong ![][flag_HK] | 5 | **6** | ![][flag_CA] **Canada** | [#1](https://osu.ppy.sh/community/matches/42048757) |
+| United States ![][flag_US] | 3 | **6** | ![][flag_TW] **Taiwan** | [#1](https://osu.ppy.sh/community/matches/42050275) |
+| **Japan** ![][flag_JP] | **6** | 0 | ![][flag_KR] South Korea | [#1](https://osu.ppy.sh/community/matches/42051882) |
+| **Indonesia** ![][flag_ID] | **6** | 2 | ![][flag_AU] Australia | [#1](https://osu.ppy.sh/community/matches/42061047) |
+| **Canada** ![][flag_CA] | **6** | 1 | ![][flag_ES] Spain | [#1](https://osu.ppy.sh/community/matches/42069729) |
 
 ### Quarterfinals
 
-| Saturday, 2018-04-21 | | | | |
-| ---: | :---: | :---: | :--- | :---: |
-| Argentina ![][flag_AR] | 0 | 5 | ![][flag_HK] Hong Kong | [#1](https://osu.ppy.sh/community/matches/41838646) |
-| Poland ![][flag_PL] | 5 | 2 | ![][flag_FR] France | [#1](https://osu.ppy.sh/community/matches/41840557) |
-| United Kingdom ![][flag_GB] | 0 | 5 | ![][flag_BR] Brazil | [#1](https://osu.ppy.sh/community/matches/41842794) |
-| Spain ![][flag_ES] | 1 | 5 | ![][flag_US] United States | [#1](https://osu.ppy.sh/community/matches/41844527) |
+| Saturday, 2018-04-21 |  |  |  |  |
+| --: | :-: | :-: | :-- | :-: |
+| Argentina ![][flag_AR] | 0 | **5** | ![][flag_HK] **Hong** Kong | [#1](https://osu.ppy.sh/community/matches/41838646) |
```

Comment:
```suggestion
| Argentina ![][flag_AR] | 0 | **5** | ![][flag_HK] **Hong Kong** | [#1](https://osu.ppy.sh/community/matches/41838646) |
```

Reply:
this exact same thing happens at least once in 2017 so I suspect some find/replace method failed